spec/commands/v2/put_content_spec.rb

```diff
@@ -221,5 +221,32 @@
         end
       end
     end
+
+    context "when no update_type is provided" do
+      before do
+        payload.delete(:update_type)
+      end
+
+      it "should send an alert to Airbrake" do
+        # swallow error about missing schema
+        expect(Airbrake).to receive(:notify)
+          .with(anything, parameters: a_hash_including(schema_path: anything))
+
+        expect(Airbrake).to receive(:notify)
+          .with(anything, parameters: a_hash_including(content_id: content_id))
+
+        described_class.call(payload)
+      end
+    end
+
+    context "when an update type is provided" do
+      it "should not send an alert to Airbrake" do
+        # swallow error about missing schema
+        expect(Airbrake).to receive(:notify)
```

Comment:
I will add an explicit `.not to` for the update type `Airbrake.notify` though which should make things clearer.